code/modules/modular_computers/file_system/programs/system/file_browser.dm

```diff
@@ -224,7 +226,8 @@
 						"name" = F.filename,
 						"type" = F.filetype,
 						"size" = F.size,
-						"undeletable" = F.undeletable
+						"undeletable" = F.undeletable,
+						"encrypted" = !!F.password
```

Comment:
My fault.
This is for the USB files. It should have been added at the normal files.
Line 214-219 atm.